lib/ddtrace/contrib/rails/action_view.rb

```diff
@@ -34,17 +34,39 @@ def self.get_key(f)
           'datadog_actionview_' + f
         end
 
-        def self.start_render_template(*)
-          key = get_key('render_template')
-          return if Thread.current[key]
+        def self.start_render_template(_name, _start, _finish, _id, payload)
+          # retrieve the tracing context
+          tracing_context = payload.fetch(:tracing_context)
```

Comment:
we call `start_render_template` only from our patched method, so `:tracing_context` key must be in the `payload`.